conf/pipeline/sarek/cfc.config

```diff
@@ -0,0 +1,24 @@
+// Profile config names for nf-core/configs
+
+params {
+  // Specific nf-core/configs params
+  config_profile_contact = 'Friederike Hanssen (@FriederikeHanssen)'
+  config_profile_description = 'nf-core/sarek CFC profile provided by nf-core/configs'
+}
+
+// Specific nf-core/sarek process configuration
+process {
+  withName:'StrelkaSingle|Strelka|StrelkaBP|MantaSingle|Manta' {
+    memory = 128.GB
+    cpus = 16
+  }
+  withName:'MSIsensor_scan|MSIsensor_msi' {
+    memory = 128.GB
+  }
+  withName:BamQC {
+    memory = 372.GB
+  }
+  withName:MapReads{
+    cpus = 48
```

Comment:
```suggestion
    cpus = 20
```

Reply:
Sure, that way we keep it really more on the lower end. I will then also reduce the memory to 60GB. Since `retry` now works, for people with bigger datasets it should automatically resubmit with double the amount then.